src/cljs/rems/application.cljs

```diff
@@ -157,12 +156,10 @@
                                                                         :error-content (format-validation-errors application (:errors response))})
                                               (rf/dispatch [::set-validation-errors (:errors response)]))))
                                :error-handler status-modal/common-error-handler!
-                               :params {:type :application.command/submit
-                                        :application-id application-id}})
+                               :params {:application-id application-id}})
                        (status-modal/common-error-handler! response)))
           :error-handler status-modal/common-error-handler!
-          :params {:type :application.command/save-draft
-                   :application-id application-id
+          :params {:application-id application-id
```

Comment:
This would be easier to read if `:params` was the first parameter, before the `:handler`, so that it would be right next to the command URL. The same reordering need applies to all three `post!` calls in this file.